validation-test/stdlib/StringMemoryTest.swift

```diff
@@ -12,6 +12,14 @@ import Foundation
 let str = "abcdefg\u{A758}hijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz\u{A759}"
 let str2 = "abcdefg\u{A759}hijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz\u{A758}"
 
+
+@inline(never)
+func getMemoryUsage() -> Int {
+  var usage = rusage()
+  getrusage(RUSAGE_SELF, &usage)
```

Comment:
This wont work on Windows, and should use `GetProcessMemoryInfo` there.

Reply:
This is not a new test - the PR just moves this code within the test.

I think this test is disabled on windows by requiring: ``// REQUIRES: objc_interop  ``